docs/getting-started-guides/ubuntu/index.md

```diff
@@ -27,6 +27,13 @@ sudo snap install conjure-up --classic
 conjure-up kubernetes
 ```
 
+As well as Homebrew for macOS:
+
+```
+brew install conjure-up
```

Comment:
I've noticed the most recent brew installs have started running brew update before it does an install, fwiw.